eng/common/scripts/Verify-Links.ps1

```diff
@@ -145,6 +145,10 @@ function CheckLink ([System.Uri]$linkUri)
     catch {
       $statusCode = $_.Exception.Response.StatusCode.value__
 
+      if(!$statusCode) {
+        $statusCode = $_.Exception.InnerException.ErrorCode
```

Comment:
Can you also add a comment here that the ErrorCode is common when there is a SocketExeception and so we are try to pull the ErrorCode from the inner SocketException if it exists.

Reply:
```suggestion
        #Pulling the error code from the inner socket exception
        $statusCode = $_.Exception.InnerException.ErrorCode
```